python/lsst/meas/algorithms/loadIndexedReferenceObjects.py

```diff
@@ -136,3 +140,50 @@ def _trim_to_circle(self, catalog_shard, ctrCoord, radius):
             if record.getCoord().separation(ctrCoord) < radius:
                 temp_cat.append(record)
         return temp_cat
+
+
+class LoadPanstarrsObjectsTask(LoadIndexedReferenceObjectsTask):
+    """Reference catalog loader for ps1_pv3_3pi_20170110 catalog
+
+    This catalog is in the "LSST format" suitable for use with
+    the `LoadIndexedReferenceObjectsTask`, but has a slightly
+    different format for the proper motion columns than was
+    eventually adopted for the default of the LSST format:
+
+    - The proper motions are in units of arcsec/yr instead of mas/yr.
+    - The epoch is in seconds since the Unix epoch instead of MJD.
```

Comment:
I'm concerned about the creation of a ReferenceObjects class designed for exactly one catalog. I worry that we'll end up with similar `LoadGaiaDR1FirstCutTask`, etc. We should probably at least call it `LoadPanstarrs20170110ObjectsTask` or something, to make it clear that it's for "one very specific PS catalog".

Could we instead write something to modify those catalogs to match the above format instead?

Reply:
I'm fine with having a `LoadPanstarrsObjectsTask`, but it seems like this is a good opportunity to write the accepted LSST one as well (although that can be a separate ticket).

In this future ticket, one might imagine making `LoadPanstarrsObjectsTask` and `LoadLSSTObjectsTask` (naming?) inherit from something slightly more general.

Reply:
Please, everyone read the docstring, which explains the motivation behind this class: it only exists to provide backward compatibility.

`LoadLsstObjectsTask` won't be necessary because we will create catalogs with the correct schema.

Reply:
@PaulPrice Ah, yes.  I understand.  Thank you for the clarification.